modules/ztp-du-host-networking-requirements.adoc

```diff
@@ -0,0 +1,83 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-du-host-networking-requirements_{context}"]
+= Distributed unit host networking requirements
+
+The following tables provide a high level overview of the networking information and custom resources required by Red Hat Advanced Cluster Management (ACM) to provision a DU bare-metal host:
+
+.Required `AgentClusterInstall` networking fields
+[cols="2,4", width="90%", options="header"]
+|====
+|Field
+|Description
+
+|`clusterNetwork`
+| Used to allocate an IPv4 or IPv6 IP address to each node. Ensure there is no overlap with `serviceNetwork`.
+
+|`serviceNetwork`
+| Block of IPv4 or IPv6 IP addresses used for cluster services internal communication in {product-title}.  Ensure there is no overlap with `clusterNetwork`.
+
+|`machineNetwork`
+| Represents the network range for external communication. Also used to determine the API and Ingress VIP addresses for provisioning the cluster.
+|====
+
+[NOTE]
+====
+You do not need to specify API and Ingress VIP addresses for DU single node clusters. Instead, when the host is provisioned by the assisted installer service, the `machineNetwork` field in the `AgentClusterInstall` CR is used to determine the API and Ingress VIP addresses.
```

Comment:
Please double-check this.

> ! need to verb
Use "need to" only when "must" is too restrictive. For example, "you do not need to" is not the same as "you must not".